README.md

```diff
@@ -15,6 +15,8 @@ Install & Upgrade Tests |
  [![Platform Advanced Installation](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-advanced.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-advanced.yml) 
  [![Platform Postgres Installation](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-postgres.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-install-postgres.yml) 
  [![Platform Upgrade](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-upgrade.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/platform-upgrade.yml) |
+[openmrs core sonaqube](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/LU-LU) 
+[openmrs sonaqube dashboard](https://sonar.openmrs.org/dashboard?id=org.openmrs%3Aopenmrs) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/LU-LU) |
```

Comment:
do we really need two budges for the sonaqube?

Reply:
Yes we need them, these are not badges, one is referring to openmrs  core sonaqube ci and  and another one is for sonaqube dashboardPage

Reply:
do we need to have sonaqube dashboard here? isn't openmrs core sonaqube ci good enough?